src/dbt/kipptaf/models/powerschool/base/base_powerschool__student_enrollments.sql

```diff
@@ -186,7 +186,7 @@ select
         )
         + 1
     ) as salesforce_graduation_year,
-
+    if(ar.region = 'Paterson', ar.prevstudentid, null) as prevstudentid,
```

Comment:
@GabyRangelB per your comment, this is where that should go. should add `student_number` instead of `null` for the `else` though